src/common/transformations/src/transformations/common_optimizations/mark_exp_reduceop_to_keep_in_mixed_precision.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/mark_exp_reduceop_to_keep_in_mixed_precision.hpp"
+
+#include <ngraph/pattern/op/wrap_type.hpp>
```

Comment:
Please use openvino headers instead of ngraph ones. Same comment for other files

Reply:
done: removed ngraph from all changed files